testsuite/run_cim_operations.py

```diff
@@ -6644,6 +6657,8 @@ def parse_args(argv_):
         elif argv[1] == '--yamlfile':
             args_['yamlfile'] = argv[2]
             del argv[1:3]
+        elif argv[1] == '-log':
```

Comment:
Why only a single hyphen in front of the `-log` option?

Reply:
Sloppy code.  Made into --log and fixed other issues. Need to rewrite the whole arg parser for this since it is homegrown and messy.